packages/cron/package.json

```diff
@@ -33,6 +33,7 @@
     "@types/node": "^16.3.1",
     "execa": "^5.1.1",
     "mocha": "^8.3.2",
-    "npm-run-all": "^4.1.5"
+    "npm-run-all": "^4.1.5",
+    "sinon": "^13.0.2"
```

Comment:
Nice, we should make sure restore the sinon sandbox at each test as stated in the docs [here](https://sinonjs.org/releases/v13/general-setup/).

[Here](https://github.com/web3-storage/web3.storage/pull/1196/commits/372d9b487b2ea2317216f48203a1704bc5a4a5fd#diff-60767f1efe42b8a4a16c4fdd5283cd62c2572192bbb11aeb1196c0bb11ed062fR61)'s an example

Reply:
Added to `hooks.js`